scripts/install_narrative.sh

```diff
@@ -122,12 +118,21 @@ then
 
     # Install sklearn and clustergrammer
     # ----------------------------------
-    pip --no-cache-dir install pandas sklearn clustergrammer_widget | tee -a "${logfile}"
+    pip --no-cache-dir install pandas scikit-learn clustergrammer_widget | tee -a "${logfile}"
```

Comment:
let's just put these mofos in one of the requirements files. Having them here just means they will be overlooked and if there's some breaking feature we will be scratching our heads for longer trying to work it out!

Reply:
I remember some vague reason in the wayback that we had to do that. Or, it could just be that we didn't adjust the requirements files right. Urgh. Yes. Let's just add them to requirements.